File: Basics_with_JavaScript/78_Conditional_Statements_Advanced/More_Exersises/03._Flowers.js
function vladi(input){

    let hrizQty = Number(input[0])
    let rosesQty = Number(input[1])
    let laleQty = Number(input[2])
    let season = (input[3])
    let command = (input[4])
    let rosesPrice = 0
    let hrizPrice = 0
    let lalePrice = 0
    let priceFlowers = 0
    switch (season) {
        case "Spring":
        case "Summer":
            hrizPrice = 2.00
            rosesPrice = 4.10
            lalePrice = 2.5
            break;
        case "Autumn":
        case "Winter":
            hrizPrice = 3.75
            rosesPrice = 4.50
            lalePrice = 4.15
            break
    }
    
    priceFlowers = hrizPrice * hrizQty + rosesPrice * rosesQty + lalePrice * laleQty
    if (command === "Y") {
        priceFlowers *= 1.15
    }
    if (season === "Spring" && laleQty > 7) {
        priceFlowers *= 0.95
    } else if (season === "Winter" && rosesQty >= 10) {
        priceFlowers *= 0.90
    }
    if (laleQty + rosesQty + hrizQty > 20) {
        priceFlowers *= 0.80
    }
     priceFlowers += 2
     priceFlowers = priceFlowers.toFixed(2)
     console.log(priceFlowers);
    }
   
    vladi(["2","4","8","Spring","Y"]);
    vladi(["3","10","9","Winter","N"]);
    vladi(["10","10","10","Autumn","N"]);

File: Basics_with_Python/PythonBasicCourse/3_Conditional_Statements_Advanced/8_Conditional_Statements_Advanced_Exercise_Lab/more_exercises/03._Flowers.py
hriz = int(input())
roses = int(input())
tulip = int(input())
season = input()
holiday = input()
hriz_price = 0
roses_price = 0
tulip_price = 0

if holiday == "Y":
    holiday = 1.15
elif holiday == "N":
    holiday = 1

if season == "Spring" or season == "Summer":
    hriz_price = 2 * holiday
    roses_price = 4.10 * holiday
    tulip_price = 2.50 * holiday
elif season == "Autumn" or season == "Winter":
    hriz_price = 3.75 * holiday
    roses_price = 4.50 * holiday
    tulip_price = 4.15 * holiday
bouquet_price = roses_price * roses + hriz_price * hriz + tulip_price * tulip

if tulip > 7 and season == "Spring":
    bouquet_price = bouquet_price * 0.95
if roses >= 10 and season == "Winter":
    bouquet_price = bouquet_price * 0.90
if hriz + roses + tulip > 20:
    bouquet_price = bouquet_price * 0.80
bouquet_price = bouquet_price + 2
print(f"{bouquet_price:.2f}")
